src/plugins/hosts/keymetaformatting.c

```diff
@@ -48,7 +48,9 @@ ssize_t keySetStringF (Key * key, const char * format, ...)
 	key->dataSize = elektraStrLen (key->data.c);
 	set_bit (key->flags, KEY_FLAG_SYNC);
 
-	return key->dataSize;
+	return key->dataSize;*/
+
+	return keySetRaw (key, p, elektraStrLen (p));
```

Comment:
Is there any reason why `keySetMeta` previously did not use `keySetRaw` to set the value, and instead reimplented part of the logic?